services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -388,13 +379,16 @@ async def get_containers_name(
 )
 @cancel_on_disconnect
 async def inspect_container(
-    request: Request, id: str, shared_store: SharedStore = Depends(get_shared_store)
+    request: Request,
+    container_id: str = PathParam(..., alias="id"),
```

Comment:
You could also patch the "thin client " inside the dynamics-sidecar to get rid of the alias. Since director-v2 and dy-sidecar are tied together as implementation.

Reply:
Fine with me, I just modify what I had.